Firestore/core/src/firebase/firestore/core/view_snapshot.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2019 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef FIRESTORE_CORE_SRC_FIREBASE_FIRESTORE_CORE_VIEW_SNAPSHOT_H_
+#define FIRESTORE_CORE_SRC_FIREBASE_FIRESTORE_CORE_VIEW_SNAPSHOT_H_
+
+namespace firebase {
+namespace firestore {
+namespace core {
+
+enum class DocumentViewChangeType { Removed, Added, Modified, Metadata };
```

Comment:
The objc assigns 0 to Removed. I realize that's the default, but may be worth porting anyways, since (according to the objc docstring) the order is relevant. Having the `=0` would help clue in readers who don't read the docstring. (I've always interpreted enums with =0 to imply order is significant and without =0 to imply it probably isn't. But that's hardly a universal interpretation.)